source/common/stream_info/stream_info_impl.h

```diff
@@ -185,6 +185,18 @@ struct StreamInfoImpl : public StreamInfo {
 
   uint64_t bytesReceived() const override { return bytes_received_; }
 
+  void addBytesRetransmitted(uint64_t bytes_retransmitted) override {
```

Comment:
I'm tentatively Ok with this but do we plan on any other HTTP/3 specific stats?  If so I'd suggest we stick them in an optional struct and lazy initialize the memory.

Reply:
No further H/3 stats planned; this completes the implementation of the [QuicAckListenerInterface](https://github.com/google/quiche/blob/main/quiche/quic/core/quic_ack_listener_interface.h). Furthermore I hope to generalize these to H/1 and H/2.